vador/src/main/java/com/salesforce/vador/execution/strategies/FailFastStrategies.kt

```diff
@@ -90,7 +97,7 @@ internal fun <FailureT, ValidatableT> failFastForEach(
   findAndFilterInvalids(
       validatables,
       failureForNullValidatable,
-      batchValidationConfig.findAndFilterDuplicatesConfigs
+      batchValidationConfig.findAndFilterDuplicatesConfigs,
```

Comment:
Pls Remove this formatting changes

Reply:
I ran spotlessApply locally and it didnt append these commas, please revert them and try that command again. Let me know if you see them agian

Reply:
Done